core/src/main/kotlin/cache/data/ChannelData.kt

```diff
@@ -78,6 +83,11 @@ public data class ChannelData(
                 defaultSortOrder,
                 totalMessageSent,
                 defaultForumLayout,
+                availableTags,
+                appliedTags,
+                defaultReactionEmoji,
+                defaultThreadRateLimitPerUser,
+                message.map { MessageData.from(it) }
```

Comment:
There is no message object as far as I'm looking at the channel object on the docs
you may be looking for the last_message_id which we already have available?

The only additional field will be the tags and the reaction emoji fields

Reply:
Message object retuns in DiscordChannel (??) https://discord.com/developers/docs/resources/channel#start-thread-in-forum-channel